tests/golden/params/argocd/argocd/25_hooks/hooks.yaml

```diff
@@ -86,7 +86,7 @@ spec:
           env:
             - name: HOME
               value: /home
-          image: mymirror.io/kubectl:1.28.4@sha256:9aa77d80cf5d32e0345c8468a45d39134a8016e30eccddaf720bf197ad7dd9f0
+          image: docker.io/bitnami/kubectl:1.28.4@sha256:9aa77d80cf5d32e0345c8468a45d39134a8016e30eccddaf720bf197ad7dd9f0
```

Comment:
For this test case the kubectl registry is overridden through field `image`, which doesn't have an effect anymore. I've intentionally left the test like this to verify the new behavior and diagnostic message.